src/applications/edu-benefits/complaint-tool/reducers/schoolSelect.js

```diff
@@ -0,0 +1,110 @@
+import {
+  LOAD_SCHOOLS_STARTED,
+  LOAD_SCHOOLS_SUCCEEDED,
+  // LOAD_SCHOOLS_FAILED,
```

Comment:
Not sure how we want to handle API failures.

Reply:
I made a separate ticket for handling this. I'm going to show the no results view if there is an API error.